Package.swift

```diff
@@ -1065,7 +1065,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // dependency version changes here with those projects.
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.4.0")),
         .package(url: "https://github.com/swiftlang/swift-driver.git", branch: relatedDependenciesBranch),
-        .package(url: "https://github.com/apple/swift-crypto.git", .upToNextMinor(from: "3.0.0")),
+        .package(url: "https://github.com/apple/swift-crypto.git", "3.0.0" ..< "3.13.0"),
```

Comment:
I’d go so far that this should be a `from` statement

Reply:
Yeah, IIRC this would exclude 3.13.1 for example

Reply:
Yes, I want to make the change as close to the original while also opening up to include the latest current release. As I said in the description, would `upToNextMajor` (of `from:`) be best here? I assumed there was a reason it's not that already 🤔